webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -214,43 +233,48 @@ describe('ContributionForm.vue', () => {
           postTitleInput = wrapper.find('.ds-input')
           postTitleInput.setValue(postTitle)
           await wrapper.vm.updateEditorContent(postContent)
-          categoryIds = ['cat12']
-          wrapper.find(CategoriesSelect).vm.$emit('updateCategories', categoryIds)
+          wrapper.find(CategoriesSelect).setData({ categories })
+          wrapper
+            .findAll('li')
+            .at(1)
+            .trigger('click') // language
+          await wrapper
+            .find(CategoriesSelect)
+            .findAll('button')
+            .at(1)
+            .trigger('click')
```

Comment:
I think we can improve this! what if someone adds more buttons? it would be nice to do similar to what @Mogge did here https://github.com/Human-Connection/Human-Connection/blob/master/webapp/components/FilterPosts/FilterPosts.spec.js#L114 ,don't you think?